src/components/custom/_multi-width-column-download.tsx

```diff
@@ -75,6 +76,8 @@ const StyledItemsWrapper = styled.div`
 `
 
 const DownloadColumn = ({ items, QRImage, QRHeading1, QRHeading2, is_rtl }: TDownloadColumn) => {
+    const renderableItems = items.filter((item) => item.visibility)
+    const hasVisibilityFalse = items.some((item) => !item.visibility)
```

Comment:
A bit of a double-negative. Can we find a better way to describe what this means?

Reply:
As checked with @fasihali-deriv , we need this `containsHiddenItem` from an array. Anything false we need to hide a text line. We can rename the variable to `containsHiddenItem` @fasihali-deriv